pkg/openshift/certgen/unstable/templates/master/tmp/bootstrapconfigs/infra-config.yaml

```diff
@@ -5,19 +5,34 @@ dnsRecursiveResolvConf: /etc/origin/node/resolv.conf
 dnsDomain: cluster.local
 dnsIP: 172.17.0.1
 dockerConfig:
-  execHandlerName: ""
+  dockerShimRootDirectory: /var/lib/dockershim
+  dockerShimSocket: /var/run/dockershim.sock
+  execHandlerName: native
 iptablesSyncPeriod: "30s"
 imageConfig:
   format: IMAGE_PREFIX/IMAGE_TYPE-${component}:${version}
   latest: False
 kind: NodeConfig
 kubeletArguments:
+  pod-manifest-path:
+  - /etc/origin/node/pods
+  bootstrap-kubeconfig:
+  - /etc/origin/node/bootstrap.kubeconfig
+  feature-gates:
+  - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+  rotate-certificates:
+  - "true"
+  cert-dir:
+  - /etc/origin/node/certificates
+  enable-controller-attach-detach:
+  - 'true'
   node-labels:
+  - node-role.kubernetes.io/infra=true
```

Comment:
hopefully you should have to update test "should label nodes correctly" in test/e2e/openshift/openshift_test.go

Reply:
Needs to run first :)

Reply:
@kwoodson bump

Reply:
Sorry I didn't understand this at first.  I've added it.